src/main/java/com/robertx22/age_of_exile/database/data/profession/items/CraftedEnchantItem.java

```diff
@@ -49,12 +51,12 @@ public CraftedEnchantItem(SlotFamily fam, String rar) {
     public void appendHoverText(ItemStack pStack, @Nullable Level pLevel, List<Component> l, TooltipFlag pIsAdvanced) {
         var tier = LeveledItem.getTier(pStack);
         l.add(TooltipUtils.tier(tier.tier));
-        l.add(Component.literal("Levels " + tier.levelRange.getMinLevel() + "-" + tier.levelRange.getMaxLevel()));
+        l.add(Gui.ENCHANTMENT_LEVEL.locName().append(tier.levelRange.getMinLevel() + "-" + tier.levelRange.getMaxLevel()));
     }
 
     @Override
     public String locNameForLangFile() {
-        return StringUTIL.capitalise(rar) + " " + fam.name() + " Enchantment";
+        return getRarity().textFormatting() + StringUTIL.capitalise(rar) + " " + fam.name() + " Enchantment";
```

Comment:
I'm not sure about the rarity color format if the color format should be added to the Enchantment items, considering you didn't add.